stratum/hal/lib/common/yang_parse_tree.cc

```diff
@@ -146,6 +146,18 @@ void YangParseTree::SendNotification(const GnmiEventPtr& event) {
 ::util::Status YangParseTree::ProcessPushedConfig(
     const ConfigHasBeenPushedEvent& change) {
   absl::WriterMutexLock r(&root_access_lock_);
+  // Make sure we clear the tree before we add new nodes.
+  root_.children_.clear();
+
+  // Add the minimum nodes:
+  //   /interfaces/interface[name=*]/state/ifindex
+  //   /interfaces/interface[name=*]/state/name
+  //   /interfaces/interface/...
+  //   /
+  // The rest of nodes will be added once the config is pushed.
+  AddSubtreeAllInterfaces();
+  AddSubtreeAllComponents();
+  AddRoot();
```

Comment:
Also called later https://github.com/stratum/stratum/pull/178/files#diff-c4a583ef9a466f4a5199f2fe94500d9fR229
Do we need both?

Reply:
good catch, I think we can put them together, I'll remove the line 229